src/index.js

```diff
@@ -1,26 +1,104 @@
+import { handleDragStart, handleDrop, cancelDefault } from './services/sort';
+import {
+  updateStatus, getTodos, saveToStorage, orderAndSave,
+} from './services/update';
+import {
+  createTodo, clearAll, clearList, editTodoInStorage,
+  editTodoInList, deleteTodoFromStorage, deleteTodoFromList,
+} from './services/todoService';
 import './style.css';
 
-const element = document.getElementById('todos-list');
-
-const myTasks = [
-  { description: 'To buy veggies', index: 0, completed: false },
-  { description: 'To meet Erick', index: 1, completed: false },
-  { description: 'To complete Microverse Capstone', index: 2, completed: false },
-];
-
-const addList = (todosList) => {
-  for (let i = 0; i < todosList.length; i += 1) {
-    const listItem = ` <div class="d-flex justify-content-between ps-3 pe-3 align-items-center">
-        <div class="d-flex">
-        <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault${i}">
-        <div class="ps-3">${todosList[i].description}</div>
-        </div>
-        <i class="fas fa-ellipsis-v text-secondary" id=""></i>
-      </div>
-      <hr>`;
-
-    element.innerHTML += listItem;
+class TodoList {
+  constructor() {
+    this.todos = getTodos();
   }
-};
 
-addList(myTasks);
\ No newline at end of file
+  handleCompletedStatus = (e) => {
+    const todos = getTodos();
+    updateStatus(e, todos);
+  }
+
+  handleEditTodo = (e) => {
+    if (e.keyCode === 13) {
+      const { value } = e.target;
+      const index = parseInt(e.target.dataset.index, 10) - 1;
+      editTodoInStorage(value, index);
+      editTodoInList(value, index);
+    }
+  }
+
+  handleDeleteTodo = (e) => {
+    const index = parseInt(e.target.dataset.index, 10) - 1;
+    deleteTodoFromList(index);
+    deleteTodoFromStorage(index);
+  }
+
+  handleClearAllCompleted = () => {
+    const todos = getTodos();
+    const filteredTodos = clearAll(todos);
```

Comment:
:heavy_check_mark: Good job for making a professional repository. :+1: 
:heavy_check_mark: The features of this milestone too have been well implemented.

We just need to fix some bugs here and there. Please see below.

- [ ] Firstly, let's format the README properly. Let each new section start on a new line for better readability. Currently, everything is muddled together and not readable. 
- [ ] I could add a new todo to the list but when I edited one todo and refreshed the page, the particular todo I edited is duplicated with the old item. In effect, I now have the newly edited todo as well as the old one. 
- [ ] When I deleted a todo and refreshed the page, the todo re-appears again while another todo that I didn't mean to delete (possibly occupying the old todo index) gets removed.
- [ ] In the same vein, I cannot completely delete all the todos, one last todo remains irremovable after each page refresh.

Kindly lets fix these bugs.

Reply:
Hi I am may I please have help with the changes you required me to do apart from the README.md.